<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Product Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>check-mark.png</normaloff>check-mark.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>768</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>1280</width>
     <height>800</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="AppTitle">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Product Sans Medium</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Recorder Inspection Monitor</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Initialization</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0" columnstretch="1,1,0">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <property name="horizontalSpacing">
           <number>20</number>
          </property>
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="6" column="2">
           <widget class="QPushButton" name="OkButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>200</height>
             </size>
            </property>
            <property name="text">
             <string>Confirm</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="ComboCam">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Select COM port</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Select Camera Device</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QComboBox" name="ComboPort">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Program Mode</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
          <item>
           <widget class="QPushButton" name="AutoButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>100</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Product Sans</family>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>AUTO</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ManualButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>100</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Product Sans</family>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>MANUAL</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="maximumSize">
       <size>
        <width>1920</width>
        <height>1080</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>80</height>
            </size>
           </property>
           <property name="title">
            <string>Camera Command</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="StartButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Product Sans</family>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>START</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="StopButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>100</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Product Sans</family>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>STOP</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>80</height>
            </size>
           </property>
           <property name="title">
            <string>Holder Command</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="GripButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Product Sans</family>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>GRIP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="RotateButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>100</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Product Sans</family>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>ROTATE</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CaptureButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Product Sans</family>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Capture</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>600</height>
            </size>
           </property>
           <property name="title">
            <string>Feature Selection</string>
           </property>
           <widget class="QCheckBox" name="GRcheck">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>40</y>
              <width>76</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Grayscale</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="NRcheck">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>70</y>
              <width>131</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Noise Reduction</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="Histocheck">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>100</y>
              <width>111</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Histogram Eq</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_7">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>130</y>
              <width>111</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Roi Extraction</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_8">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>190</y>
              <width>121</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Blob Analysis</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_9">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>160</y>
              <width>131</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Template Matching</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="maximumSize">
          <size>
           <width>1500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="OriCamlabel">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1" rowspan="2">
           <widget class="QLabel" name="ProcessedCapturedLabel">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_6">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="title">
             <string>Program Command | Status</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="1" rowspan="2">
              <widget class="QPushButton" name="StoppButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Product Sans</family>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>STOP</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" rowspan="2">
              <widget class="QPushButton" name="InspectButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Product Sans</family>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>INSPECT</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QProgressBar" name="progressBar">
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="statsLabel">
               <property name="font">
                <font>
                 <family>Product Sans Black</family>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="text">
                <string>STATUS: </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
               <property name="indent">
                <number>2</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="ProcessedCamlabel">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGuide"/>
    <addaction name="actionAbout_Software"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionGuide">
   <property name="text">
    <string>Guide</string>
   </property>
  </action>
  <action name="actionAbout_Software">
   <property name="text">
    <string>About Software..</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
